dot/sync/chain_sync.go

```diff
@@ -161,6 +163,8 @@ type chainSync struct {
 	minPeers         int
 	maxWorkerRetries uint16
 	slotDuration     time.Duration
+
+	logSyncSpeedFrequency time.Duration
```

Comment:
nit rename to `logSyncPeriod`? Since it's more of a period, not a frequency :thinking:

Reply:
Updated.